### PR TITLE
Initial operator version

### PR DESCRIPTION
**Pull request by jwerak**

### CODE REVIEW

File: operator/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+
+RUN pip install kubernetes==8.0
```
Comment (tomkukral) [inline operator/Dockerfile]: I'd prefer to install from `requirementes.txt` to make local development easier.

File: operator/deploy/operator.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    metadata:
+      labels:
+        app: pycont-operator
+    spec:
+      serviceAccountName: pycont-operator
+      containers:
+      - image: prgcont/pycont-operator
+        name: pycont-operator
+        imagePullPolicy: Always
+        command:
```
Comment (tomkukral) [inline operator/deploy/operator.yml]: Why are you changing command when it's aleady defined in `Dockerfile`?

File: operator/operator.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+from kubernetes import client, config
+from datetime import datetime, timedelta
+import time
+import requests
+import os
+from requests.auth import HTTPBasicAuth
+
+MY_NAMESPACE = "pycont"
```
Comment (tomkukral) [inline operator/operator.py]: Read it from env variable as well?

File: operator/operator.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+
+def check_latency(namespace, name, latency_treshold):
+    # TODO check latency for this app
+    prom_url = "http://pyvo.prgcont.cz/prometheus/api/v1/query"
+    query_data = {
+        "query": 'http_requests_total{job="kubernetes-node-kubelet", handler="prometheus"}[5m]'
+    }
+    r = requests.post(
+        prom_url, data=query_data, auth=HTTPBasicAuth(PROM_USER, PROM_PASSWORD)
```
Comment (tomkukral) [inline operator/operator.py:74]: Please don't share password which you got. I'll give you another login just for operator

**Review by tomkukral — COMMENTED** [review]
I'd prefet to have importable library but it's fine for the first version.

File: operator/operator.py (hunk 4 above)
Comment (jwerak) [reply operator/operator.py:74]: gimme ;)

**Review by tomkukral — APPROVED** [review]

**Pull request merged**